### PR TITLE
Add JPEG-Archive (new formula)

### PR DESCRIPTION
**Pull request by danielgtaylor**
This adds my [JPEG-Archive](https://github.com/danielgtaylor/jpeg-archive) project to Homebrew.

There is also a precompiled binary available on the releases page, but I wasn't sure how to set up Homebrew to use it. This formula compiles the latest release and depends on `mozjpeg`.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
A few audits failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24604/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_jpeg_archive___strict/

File: Library/Formula/jpeg-archive.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on "mozjpeg"
+
+  def install
+    system "make"
+    bin.install "jpeg-archive", "jpeg-recompress", 'jpeg-hash', 'jpeg-compare'
+  end
+
+  test do
+    system "#{bin}/jpeg-recompress",
+           "/System/Library/CoreServices/DefaultDesktop.jpg",
```
Comment (mikemcquaid) [inline Library/Formula/jpeg-archive.rb]: Use `test_fixtures("test.jpg")` instead

File: Library/Formula/jpeg-archive.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+class JpegArchive < Formula
+  homepage "https://github.com/danielgtaylor/jpeg-archive"
+  url "https://github.com/danielgtaylor/jpeg-archive/archive/2.1.1.tar.gz"
+  sha1 "874846fdc3a44811cccec6415ddf4d46c0fcd568"
+
+  depends_on "mozjpeg"
+
+  def install
+    system "make"
+    bin.install "jpeg-archive", "jpeg-recompress", 'jpeg-hash', 'jpeg-compare'
```
Comment (mikemcquaid) [inline Library/Formula/jpeg-archive.rb]: Please use the `make install` instead.

**Comment by danielgtaylor** [comment]
@mikemcquaid thanks for the feedback! Formula has been updated; let me know if you need anything else.

File: Library/Formula/jpeg-archive.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      PREFIX=#{prefix}
+    ]
+
+    system "make", "install", *args
+  end
+
+  test do
+    system "#{bin}/jpeg-recompress",
+           test_fixtures("test.jpg"),
+           "output.jpg"
```
Comment (mikemcquaid) [inline Library/Formula/jpeg-archive.rb]: Maybe just put these all on one line?

File: Library/Formula/jpeg-archive.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://github.com/danielgtaylor/jpeg-archive"
+  url "https://github.com/danielgtaylor/jpeg-archive/archive/2.1.1.tar.gz"
+  sha256 "494534f5308f99743f11f3a7c151a8d5ca8a5f1f8b61ea119098511d401bc618"
+
+  depends_on "mozjpeg"
+
+  def install
+    args = %W[
+      PREFIX=#{prefix}
+    ]
```
Comment (mikemcquaid) [inline Library/Formula/jpeg-archive.rb]: I'd just inline this variable, I don't see the advantage of splitting it out.

**Comment by danielgtaylor** [comment]
@mikemcquaid feedback applied, please have another look!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `jpeg-archive 2.1.1 (new formula)`.

**Pull request closed without merging**